src/timeMachine/components/TimeMachine.tsx

```diff
@@ -26,11 +26,15 @@ const TimeMachine: FunctionComponent = () => {
     'time-machine--split': isViewingVisOptions,
   })
 
-  let bottomContents: JSX.Element = null
-  if (activeTab === 'alerting') {
-    bottomContents = <TimeMachineAlerting />
-  } else if (activeTab === 'queries') {
-    bottomContents = <TimeMachineQueries />
+  const [bottomRef, setBottomRef] = useState(null)
```

Comment:
There's a [`useRef`](https://reactjs.org/docs/hooks-reference.html#useref) hook that does this for us